.github/workflows/ci.yml

```diff
@@ -79,25 +79,33 @@ jobs:
         if: matrix.os != 'macos-latest'
         run: make test
 
+      # Store a cnd ${{ matrix.os }} archive that contains the cnd binary
```

Comment:
Do we need this comment? 
It literally says the same thing one line below 😅

Reply:
I added it because I confused myself earlier because I through that the artefact would be the cnd binary, but the artefact is a zip that contains the cnd binary (actually being named `cnd`). If you feel that is clear anyway then I can remove it.

Reply:
You could just update the step name to include the fact that it is an archive :)

Reply:
done